components/UI/desktopNav.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FunctionComponent, MouseEvent } from "react";
+import styles from "../../styles/components/desktopNav.module.css";
+import Link from "next/link";
+import { FaDiscord, FaGithub, FaTwitter } from "react-icons/fa";
+import theme from "../../styles/theme";
+
+const DesktopNav: FunctionComponent = () => {
```

Comment:
On this component, I would like something where I can click outside the box and the box closes.

<img width="473" alt="Capture d’écran 2023-11-12 à 16 45 17" src="https://github.com/starknet-id/app.starknet.id/assets/78437165/b918c2b7-0a37-4a63-81b8-f979551c3f4b">